openshift/dynamic/client.py

```diff
@@ -256,6 +263,38 @@ def request(self, method, path, body=None, **params):
         )
 
 
+    def validate(self, resource, version=None, strict=False):
```

Comment:
would you mind renaming `resource` here? In the methods above `resource` is used to refer to the class of resource, not an individual instance. I think `definition` would be better.